docs/index.rst

```diff
@@ -45,3 +45,4 @@ Featured tutorials
    Getting Started<getting_started>
    Workflow SDK <sdk/index>
    Quantum SDK <quantum/index>
+   recipes/index
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Does this pick up the title from `docs/recipes/index.rst`?

Reply:
Yup the sidebar will use the title from that file